MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -2799,4 +2869,22 @@ - (MXGroup *)groupWithGroupId:(NSString*)groupId
     return group;
 }
 
+#pragma mark - MXScanManager notifications
+
+- (void)registerScanManagerNotifications
+{
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(eventScansDidChange:) name:MXScanManagerEventScanDidChangeNotification object:nil];
+}
+
+- (void)unregisterScanManagerNotifications
```

Comment:
Should we call `unregisterScanManagerNotifications` on `[self destroy]` like other registered notifications?